bin/setup.sh

```diff
@@ -12,11 +11,23 @@ printf "found!\n"
 
 # Check for installed ruby version
 printf "Checking for rbenv: "
-rbenv version >/dev/null 2>&1 || { printf >&2 "nope!\nrbenv must be installed. Try running:\n\tbrew install rbenv\n"; exit 1; }
-printf "found!\n"
-printf "Ensuring correct ruby version is installed:\n"
-rbenv install -s
-printf "Correct ruby version installed!\n"
+if rbenv version >/dev/null 2>&1 ; then
+    printf "found!\n"
+    printf "Ensuring correct ruby version is installed:\n"
+    rbenv install -s
+    printf "Correct ruby version installed!\n"
+
+else
+    printf "nope!\n"
+    ACTUAL=`ruby -v | awk '{print $2}' | awk -Fp '{print $1}' | tr -d '\n'`
+    EXPECTED=`cat .ruby-version | tr -d '\n'`
+    if [ $ACTUAL = $EXPECTED ] ; then
+	printf "Correct ruby version is already installed, without rbenv\n"
```

Comment:
I'm honestly a little wary about not blocking on `rbenv`. The nice thing about it is it pretty much removes a slew of debugging questions that aren't even really relevant to the codebase. It lets us not care about global ruby version changes or local ruby version requirement changes; system ruby usually requires root privileges to install gems; and so on. Do you have thoughts on why it should be a looser requirement?